### ISSUE TITLE
pip安装paddlepaddle失败

### ISSUE THREAD

**Issue by 930083287** [PaddlePaddle/Paddle#14052]
利用pip安装paddlepaddle，显示是paddlepaddle (1.0.2)，安装过程没有出错，但是验证是否安装成功时，发现import paddle.v2正常，而import paddle.fluid，失败
>>> import paddle.fluid
*** Aborted at 1540375982 (unix time) try "date -d @1540375982" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGSEGV (@0x50) received by PID 119914 (TID 0x7f8291d13700) from PID 80; stack trace: ***
    @     0x7f8291512390 (unknown)
    @     0x7f8291b0873c (unknown)
    @     0x7f8291b11851 (unknown)
    @     0x7f8291b0c564 (unknown)
    @     0x7f8291b10da9 (unknown)
    @     0x7f8290b6a5ad (unknown)
    @     0x7f8291b0c564 (unknown)
    @     0x7f8290b6a664 __libc_dlopen_mode
    @     0x7f8290b3ca85 (unknown)
    @     0x7f829150fa99 __pthread_once_slow
    @     0x7f8290b3cba4 backtrace
    @     0x7f8235f2ac46 paddle::platform::EnforceNotMet::EnforceNotMet()
    @     0x7f823721f246 paddle::platform::GetCUDADeviceCount()
    @     0x7f823715d936 paddle::framework::InitDevices()
    @     0x7f8235f0ba5a _ZZN8pybind1112cpp_function10initializeIZN6paddle6pybindL13pybind11_initEvEUlbE65_vIbEINS_4nameENS_5scopeENS_7siblingEEEEvOT_PFT0_DpT1_EDpRKT2_ENUlRNS_6detail13function_callEE1_4_FUNESL_
    @     0x7f8235f3b884 pybind11::cpp_function::dispatcher()
    @     0x7f8291800cc4 PyEval_EvalFrameEx
    @     0x7f82918024e9 PyEval_EvalCodeEx
    @     0x7f82917ff9b8 PyEval_EvalFrameEx
    @     0x7f82918024e9 PyEval_EvalCodeEx
    @     0x7f829180270a PyEval_EvalCode
    @     0x7f829181223e PyImport_ExecCodeModuleEx
    @     0x7f82918124bd load_source_module
    @     0x7f829181393f load_package
    @     0x7f8291813143 import_submodule
    @     0x7f8291813398 load_next
    @     0x7f8291813cec PyImport_ImportModuleLevel
    @     0x7f82917f6e26 builtin___import__
    @     0x7f82917667a3 PyObject_Call
    @     0x7f82917f8a58 PyEval_CallObjectWithKeywords
    @     0x7f82917fe458 PyEval_EvalFrameEx
    @     0x7f82918024e9 PyEval_EvalCodeEx
Segmentation fault (core dumped)

**Comment by kuke** [comment]
See https://github.com/PaddlePaddle/Paddle/issues/12610

**Comment by 930083287** [comment]
目前已经按照cuda版本和cudnn版本安装完毕，import paddle.fluid不报错，但是训练时报错。
其中执行：train_exe = fluid.ParallelExecutor(main_program=train_prog,
            use_cuda=use_gpu, loss_name=loss.name)出错

-----------  Configuration Arguments -----------
ap_version: 11point
batch_size: 64
data_dir: data/pascalvoc
dataset: pascalvoc
enable_ce: False
epoc_num: 120
image_shape: 3,300,300
learning_rate: 0.001
mean_BGR: 127.5,127.5,127.5
model_save_dir: model
parallel: True
pretrained_model: pretrained/ssd_mobilenet_v1_coco/
use_gpu: True
------------------------------------------------
/home/users/songliejin/miniconda2/lib/python2.7/site-packages/paddle/fluid/evaluator.py:71: Warning: The DetectionMAP is deprecated, because maintain a modified program inside evaluator cause bug easily, please use fluid.metrics.DetectionMAP instead.
  % (self.__class__.__name__, self.__class__.__name__), Warning)
*** Aborted at 1540433215 (unix time) try "date -d @1540433215" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGSEGV (@0x50) received by PID 126041 (TID 0x7f652f391700) from PID 80; stack trace: ***
    @     0x7f652eb90390 (unknown)
    @     0x7f652f18673c (unknown)
    @     0x7f652f18f851 (unknown)
    @     0x7f652f18a564 (unknown)
    @     0x7f652f18eda9 (unknown)
    @     0x7f652e1e85ad (unknown)
    @     0x7f652f18a564 (unknown)
    @     0x7f652e1e8664 __libc_dlopen_mode
    @     0x7f652e1baa85 (unknown)
    @     0x7f652eb8da99 __pthread_once_slow
    @     0x7f652e1baba4 backtrace
    @     0x7f64d836bde6 paddle::platform::EnforceNotMet::EnforceNotMet()
    @     0x7f64d9796e1e paddle::platform::dynload::GetNCCLDsoHandle()
    @     0x7f64d844a089 _ZSt16__once_call_implISt12_Bind_simpleIFZN6paddle8platform7dynload24DynLoad__ncclCommInitAllclIIPP8ncclCommiPiEEEDTcl15ncclCommInitAllspfp_EEDpT_EUlvE_vEEEvv
    @     0x7f652eb8da99 __pthread_once_slow
    @     0x7f64d844dd71 paddle::platform::NCCLContextMap::NCCLContextMap()
    @     0x7f64d8447e95 paddle::framework::ParallelExecutor::ParallelExecutor()
    @     0x7f64d83be295 _ZZN8pybind1112cpp_function10initializeIZNS_6detail4initIIRKSt6vectorIN5boost7variantIN6paddle8platform9CUDAPlaceENS8_8CPUPlaceENS8_15CUDAPinnedPlaceENS5_6detail7variant5void_ESE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_EESaISF_EERKSt13unordered_setISsSt4hashISsESt8equal_toISsESaISsEESS_RKNS7_9framework11ProgramDescERKSsPNST_5ScopeERS4_IS10_SaIS10_EERKNST_7details17ExecutionStrategyERKNS14_13BuildStrategyEmmEE7executeINS_6class_INST_16ParallelExecutorEIEEEIELi0EEEvRT_DpRKT0_EUlPS1E_SJ_SS_SS_SW_SY_S10_S13_S17_S1A_mmE_vIS1M_SJ_SS_SS_SW_SY_S10_S13_S17_S1A_mmEINS_4nameENS_9is_methodENS_7siblingEEEEvOT_PFT0_DpT1_EDpRKT2_ENKUlRNS2_13function_callEE1_clES23_
    @     0x7f64d83be3ee _ZZN8pybind1112cpp_function10initializeIZNS_6detail4initIIRKSt6vectorIN5boost7variantIN6paddle8platform9CUDAPlaceENS8_8CPUPlaceENS8_15CUDAPinnedPlaceENS5_6detail7variant5void_ESE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_SE_EESaISF_EERKSt13unordered_setISsSt4hashISsESt8equal_toISsESaISsEESS_RKNS7_9framework11ProgramDescERKSsPNST_5ScopeERS4_IS10_SaIS10_EERKNST_7details17ExecutionStrategyERKNS14_13BuildStrategyEmmEE7executeINS_6class_INST_16ParallelExecutorEIEEEIELi0EEEvRT_DpRKT0_EUlPS1E_SJ_SS_SS_SW_SY_S10_S13_S17_S1A_mmE_vIS1M_SJ_SS_SS_SW_SY_S10_S13_S17_S1A_mmEINS_4nameENS_9is_methodENS_7siblingEEEEvOT_PFT0_DpT1_EDpRKT2_ENUlRNS2_13function_callEE1_4_FUNES23_
    @     0x7f64d837ca24 pybind11::cpp_function::dispatcher()
    @     0x7f652ede47a3 PyObject_Call
    @     0x7f652edf363d instancemethod_call
    @     0x7f652ede47a3 PyObject_Call
    @     0x7f652ee3d584 slot_tp_init
    @     0x7f652ee39e3b type_call
    @     0x7f652ede47a3 PyObject_Call
    @     0x7f652ee7ab69 PyEval_EvalFrameEx
    @     0x7f652ee804e9 PyEval_EvalCodeEx
    @     0x7f652ee09377 function_call
    @     0x7f652ede47a3 PyObject_Call
    @     0x7f652edf363d instancemethod_call
    @     0x7f652ede47a3 PyObject_Call
Segmentation fault (core dumped)

**Comment by kuke** [comment]
看起来是nccl的报错，请问nccl是怎么安装的呢

**Comment by lucywsq** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持!
